src/types/Module.ts

```diff
@@ -78,6 +78,7 @@ export type ModuleDeliverable = {
   dueDate: DateTime;
   status: DeliverableStatusType;
   type: DeliverableTypeType;
```

Comment:
type should be `string` here and type(raw) should be `DeliverableTypeType`. But for clarity maybe we can rename `type` to `typeName` and `type(raw)` to `type`.

Reply:
I usually try to differentiate between the "search results" type and the general type. I think `ModuleDeliverable` should only have the hydrated and verified `type: DeliverableTypeType`, whereas the `ModuleDeliverableSearchResult` can have `type(raw): string` because that is specific to the search API.

Reply:
@nickgraz I agree and that's exactly the change I just made